docs/en_US/TrainingService/PaiMode.md

```diff
@@ -59,6 +59,8 @@ Compared with [LocalMode](LocalMode.md) and [RemoteMachineMode](RemoteMachineMod
     * Optional key. Set the virtualCluster of OpenPAI. If omitted, the job will run on default virtual cluster.
 * shmMB
     * Optional key. Set the shmMB configuration of OpenPAI, it set the shared memory for one task in the task role.
+* authFile
```

Comment:
why the name of this field is different from kubeflowMode?

Reply:
because the private registry logic in PAI and Kubernetes is different. In PAI, users need to pass a field authFile in job config, and submit the job, refer https://github.com/microsoft/pai/blob/12dd8d3a7379c264b337b308a320776127ed3ee4/docs/zh_CN/job_tutorial.md. in Kubernetes, users need to create a secret, and pass the secret name in job config, refer https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/